frontend/public/components/node.tsx

```diff
@@ -61,7 +63,7 @@ const Header = props => {
   </ListHeader>;
 };
 
-const NodeStatus = ({node}) => <StatusIcon status={nodeStatus(node)} />;
+//const NodeStatus = ({node}) => <StatusIcon status={nodeStatus(node)} />;
```

Comment:
leftover

Reply:
I wasn't sure if we should remove it or just comment it out :)

Reply:
All right - better for me when performing upstream merges :-)